pkg/ui/ui.go

```diff
@@ -58,20 +39,16 @@ func NewBaseUI(logger log.Logger, menuTmpl string, funcMap template.FuncMap, tmp
 func (bu *BaseUI) serveStaticAsset(w http.ResponseWriter, req *http.Request) {
 	fp := route.Param(req.Context(), "filepath")
 	fp = filepath.Join("pkg/ui/static", fp)
-	bu.serveAsset(fp, w, req)
+	_ = bu.serveAsset(fp, w, req)
```

Comment:
Don't ignore errors. If error happenes here, that means we don't have the file locally, but now we are not returning 404 either, this will result in no response to the client.

```suggestion
	if err := bu.serveAsset(fp, w, req); err != nil {
		level.Warn(bu.logger).Log("msg", "Could not get file", "err", err, "file", fp)
		w.WriteHeader(http.StatusNotFound)
	}
```